lib/matplotlib/mpl-data/kpsewhich.lua

```diff
@@ -0,0 +1,3 @@
+-- see dviread._LuatexKpsewhich
+kpse.set_program_name("tex")
+while true do print(kpse.lookup(io.read():gsub("\r", ""))) end
```

Comment:
Is it guaranteed to flush on every `print`?

Reply:
AFAICT, Lua uses standard C libraries, so may be buffered. In that case, you'd need a `io.flush` here too?

Reply:
Added a call to flush().